test/scripts/imgtestlib.py

```diff
@@ -282,11 +284,19 @@ def check_for_build(manifest_fname, build_info_path, errors):
 
         # check if it's a BIB type and compare image IDs
         if image_type in BIB_TYPES:
-            booted_id = dl_config.get("bib-id", None)
+            # Successful boot tests with BIB add a file to the directory as bib-<image ID>. Collect them and compare.
+            bib_ids = glob("bib-*", root_dir=build_info_dir)
+            # add the _old_ bib ID that we used to keep in the info.json
+            if config_bib_id := dl_config.get("bib-id"):
```

Comment:
Python 3.6 incompatible :/

Reply:
I could've sworn I noticed we use this already elsewhere.  But probably in none of the code we run on the CS8 test.

Reply:
Fixed.